### PR TITLE
Pre requisite

### PR DESCRIPTION
**Pull request by DavidGeismarLtd**

### CODE REVIEW

**Review by ManyBaby — CHANGES_REQUESTED** [review]
I think this is not why Pundit is made for.
Pundit is authorization based on who is the user and what ressources he/she is trying to access. 
You should have solved that issue with a default scope.

**Comment by DavidGeismarLtd** [comment]
> I think this is not why Pundit is made for.
> Pundit is authorization based on who is the user and what ressources he/she is trying to access.
> You should have solved that issue with a default scope.

noted

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,8 +1,15 @@
 class ApplicationController < ActionController::Base
+  include Pundit
```
Comment (ManyBaby) [inline app/controllers/application_controller.rb:2]: We will use Pundit for other things

**Review by ManyBaby — APPROVED** [review]

**Review by ManyBaby — APPROVED** [review]
On my phone I can't see if the published scope already existed on Program.
The rest seems good. Well played

**Pull request merged**